lib/c2_version.rb

```diff
@@ -13,16 +13,12 @@ def attributes
     end
   end
 
+  def live_version
```

Comment:
can we put this below the `diff` method? this is in the spirit of:

> dependencies in a given class should always point downward. That is, the first method in a class should depend on methods further down in the class definition, and those methods should depend on methods even further down, and so on.

which is courtesy of Avdi Grimm / Ruby Tapas